src/Tizen.NUI/src/public/Images/NativeImageQueue.cs

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    using global::System;
+
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NativeImageQueue : NativeImageInterface
+    {
+        private IntPtr handle;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public NativeImageQueue(uint width, uint height, NativeImageSource.ColorDepth depth) : this(Interop.NativeImageQueue.NewHandle(width, height, (int)depth), true)
```

Comment:
Adding comment of this constructor and each argument are needed.

Reply:
Thank you. I add a comment